src/test/java/com/browserstack/SingleTest.java

```diff
@@ -5,17 +5,28 @@
 
 import org.testng.Assert;
 import org.testng.annotations.Test;
+import org.openqa.selenium.JavascriptExecutor;
 
 public class SingleTest extends BrowserStackTestNGTest {
 
     @Test
     public void test() throws Exception {
         driver.get("https://www.google.com/ncr");
+        try {
+            WebElement ele = driver.findElement(By.xpath("//div[text()='I agree']"));
+            if(ele.isDisplayed()){
+                ((JavascriptExecutor)driver).executeScript("arguments[0].scrollIntoView()", ele);
+                Thread.sleep(5000);
```

Comment:
We need this `sleep` because we are scrolling inside a modal to click on a div named I agree. and scrolling takes its own time for that element to be intractable.
If we do not add a sleep then getting following error while running `click()` command.
```element click intercepted: Element <div class="jyfHyd" role="none">...</div> is not clickable at point (246, 524). ```